test/e2e_env/universal/compatibility/dp_compatibility.go

```diff
@@ -19,9 +19,9 @@ func UniversalCompatibility() {
 			Install(Kuma(core.Zone)).
 			Install(TestServerUniversal("test-server", "default",
 				WithArgs([]string{"echo", "--instance", "universal1"}),
-				WithDPVersion("1.5.0"))).
+				WithDPVersion("2.5.0"))).
 			Install(DemoClientUniversal(AppModeDemoClient, "default",
-				WithDPVersion("1.5.0"),
+				WithDPVersion("2.5.0"),
```

Comment:
Ideally we should use `versions.yml` to pick something that's 2 version behind us so that it always stays up to date.

Reply:
Yeah that's definitely true